components/accordion-menu/index.js

```diff
@@ -1,11 +1,7 @@
-/* eslint-disable linebreak-style */
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Coherent Labs AD. All rights reserved.
- *  Licensed under the MIT License. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-if (process.env.NODE_ENV === 'production') {
-    module.exports = require('./cjs/accordion-menu.production.min.js');
-} else {
-    module.exports = require('./cjs/accordion-menu.development.js');
-}
+import AccordionMenu from './script.js';
+import * as bundle from './dist/accordion-menu.development.js';
+
+export {
+    AccordionMenu,
+    bundle,

```

Comment:
Why do we need the bundle in the index.js? What is the use case here? When do I need to import it and where?

Reply:
if you want to import the raw source and use your own bundler - you import AccordionMenu.
If you want to import the bundle but not use a bundler - you import bundle.js and load the file that imports it using <script type="module">